src/components/Questions.tsx

```diff
@@ -25,7 +25,11 @@ interface QuizProps {
 const Questions: React.FC<QuizProps> = (QuizProps) => {
   return (
     <>
-      <h1 className="quiz-heading">{QuizProps.currQuestion.Question}</h1>
+      <h1 className="quiz-heading">Question {QuizProps.questionNumber}</h1>
+      <br />
+      <div className="quiz-text">
+        <p>{QuizProps.currQuestion.Question}</p>
```

Comment:
I think using the paragraph element is to small. 

I think it would be better to use an `h2`